input/blog/2020-02-01-cake-v0.37.0-released.md

```diff
@@ -0,0 +1,48 @@
+﻿---
+title: Cake v0.37.0 released
+category: Release Notes
+author: devlead
+---
+
+Version 0.37.0 of Cake has been released.
+
+In this version we've got a few improvements and one bug fix, full list of changes at end of this post.
+
+<img style="float: right;" src="https://user-images.githubusercontent.com/1647294/73600130-7c17ff80-454c-11ea-8c9a-981224c0864d.png" />
+
+🎉 But first we would like to share a small milestone, [Cake.Tool](https://www.nuget.org/packages/Cake.Tool/), the .NET Global tool version of Cake script runner, has now been downloaded over one million times from [NuGet.org](https://www.nuget.org/packages/Cake.Tool/), we've seen some real uptake lately and are really excited about that.
```

Comment:
Perhaps add a `.` after `one million times from [NuGet.org](https://www.nuget.org/packages/Cake.Tool/),` and start a new sentence?